adagrams/game.py

```diff
@@ -1,13 +1,166 @@
 from random import randint
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+# wave_01:
+# we need to create function that select 10 letters randomly.
+# make sure to return letters as list of strings.
+# draw_letter() letter selected no  ore times that its available.
+# test3 was not passing so I looked up chatgt to see why, went over it with Mikelle to understand.
 
 def draw_letters():
-    pass
+    letter_list = [] # store letters based on their frequancy
+    for letter, count in LETTER_POOL.items(): # we used .items() because we need to return a dict with letters:key, count:value
+        for i in range(count):
+            letter_list.append(letter)
+    print(letter_list)
```

Comment:
When submitting code, just make sure to remove any print statements that have been used for debugging purposes!